Automattic/pocket-casts-android
# Retrieve show notes from JSON cache

## Description

This change switches getting the show notes from the JSON cache.

## Testing Instructions
1. Open a podcast page
2. Tap on an episode row
3. ✅ Verify the show notes are displayed
4. Close the episode fragment
5. Turn on airplane mode
6. Tap another episode row
7. ✅ Verify the show notes are displayed 
8. Turn off airplane mode
9. Go to a different podcast
10. Tap play on an episode row without opening the episode fragment
11. Tap on the mini player
12. Tap on the Notes tab
13. ✅ Verify the show notes are displayed

### Test show notes are cached with an episode download
1. Tap the Profile tab
2. Tap the settings cog
3. Tap General
4. Tap Row action
5. Change it to Download
6. Go to discover and find a podcast that you haven't opened
7. Tap download on the episode row and wait for the download to finish
8. Turn on airplane mode
9. Tap on an episode row
10. ✅ Verify the show notes are displayed
